tests/subsys/mgmt/mcumgr/all_options/prj.conf

```diff
@@ -8,40 +8,40 @@ CONFIG_ZTEST_NEW_API=y
 CONFIG_TINYCRYPT_SHA256=y
 CONFIG_FILE_SYSTEM=y
 CONFIG_MCUMGR=y
-CONFIG_MCUMGR_SMP_DUMMY=y
-CONFIG_MCUMGR_SMP_DUMMY_RX_BUF_SIZE=512
-CONFIG_MCUMGR_CMD_FS_MGMT=y
-CONFIG_FS_MGMT_FILE_STATUS=n
-CONFIG_FS_MGMT_CHECKSUM_HASH=y
+CONFIG_MCUMGR_TRANSPORT_DUMMY=y
+CONFIG_MCUMGR_TRANSPORT_DUMMY_RX_BUF_SIZE=512
```

Comment:
IMO this commit should be squashed into 1st commit, so that this patch series is bisectable.